packages/react-app/src/pages/edit.tsx

```diff
@@ -553,20 +570,25 @@ const EditGiveaway = () => {
                   <Controller
                     name={'manual'}
                     control={control}
-                    defaultValue={data?.manual ? data?.manual : false}
-                    render={({ field: { ref, ...field }, fieldState }) => (
+                    defaultValue={true}
+                    render={({ field: { ref, ...field } }) => (
                       <Checkbox
                         {...field}
-                        checked={field.value}
+                        checked={true}
                         inputRef={ref}
-                        disabled={!!giveawayId}
+                        disabled={true}
                       />
                     )}
                   />
                 }
                 label="Manual giveaway"
                 disabled={!!giveawayId}
               />
+              <WarningBox
+                message="Automatic raffles will be available in the next
+                        version. The manual option requires an administrator 
+                        to raffle winners when the giveaway ends."
```

Comment:
A raffle involves buying a ticket (é uma rifa, basicamente)
```suggestion
                message="Automatic giveaways will be available in the next
                        version. The manual option requires an administrator 
                        to generate winners when the giveaway ends."
```